src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -410,7 +410,7 @@
   - name: --parameters -p
     short-summary: Supply deployment parameter values.
     long-summary: >
-        Parameters may be supplied from a file using the `@{path}` syntax, a JSON string, or as <KEY=VALUE> pairs. Parameters are evaluated in order, so when a value is assigned twice, the latter value will be used.
+        Parameters may be supplied from a file using the `@{path}` syntax, a JSON string, or as `<KEY=VALUE>` pairs. Parameters are evaluated in order, so when a value is assigned twice, the latter value will be used.
```

Comment:
Since we have chosen to replace the placeholder tag from < > to { }, should these descriptions also remain consistent?

Reply:
Most of the situation we add ` ` to wrap problematic code snippet to make it pass the doc pipeline html tag detection and remain the cmd formatting in doc page at the same time. 

Replace < > with {  } is only applied when we found adding `` ` ` `` might causing confusions to users, like the parameter formatting with quoting.

Reply:
< > is also a commonly used placeholder tag in cli doc